### DISCUSSION TITLE
Which Passbands does Phoebe need to run_compute()?

### DISCUSSION THREAD

**Discussion by rfreire-400** [phoebe-project/phoebe2#748] [Questions]
I'm running Phoebe remotely on a cluster and I'm getting the following error:

<img width="919" alt="Screenshot 2023-06-08 at 2 58 15 PM" src="https://github.com/phoebe-project/phoebe2/assets/113457579/360c5153-36c0-4dd3-921b-659f7fe1ffec">
<img width="919" alt="Screenshot 2023-06-09 at 9 15 14 AM" src="https://github.com/phoebe-project/phoebe2/assets/113457579/9d2c08e6-c325-4020-91f3-d91984b6d75a">

The code itself is a very simple run_compute() with a light curve dataset added. I didn't specify a passband for it, but I'm guessing it defaulted to Johnson:V

I don't have admin access so I cannot update openssl like the FAQ suggests. The compute nodes on the cluster do not have internet access so I wanted to manually download the necessary passbands to my computer and upload them to the cluster. Would this solve this issue? And if so, which passbands should I download? 

As of right now, it only seems to have 'Bolometric:900-40000' and 'Johnson:V' installed (I'm guessing by default).

**Comment by kecnry** [comment]
This looks like it is just a warning when phoebe attempts to get the _list_ of available passbands online.  So long as your datasets all use the default passband (or any other passband that is available locally), you should be able to ignore this warning and `run_compute()` should succeed without errors.  Since this is retrying and failing though, this will add some overhead every time you import phoebe... to avoid that, set the `PHOEBE_ENABLE_ONLINE_PASSBANDS=FALSE` [environment variable before importing phoebe](http://phoebe-project.org/docs/2.4/tutorials/envars).

If you need passbands that are not installed by default, you can [download them manually from here](http://phoebe-project.org/tables/pbs) and copy them to the server.  To see the appropriate directories, call `phoebe.list_passband_directories()` on the server installation.  If the bundle was initialized on the server without the full list of passbands available and then you add local files, the options in the passband parameter may not update... if that's the case, let us know and we can point you send you a snippet of code to update those choices to the files installed locally.

**Reply by rfreire-400** [reply]
Setting PHOEBE_ENABLE_ONLINE_PASSBANDS=FALSE stopped the warnings and I was able to run the code. Turns out there was an issue with the way I was importing TESS data on the cluster and I didn't notice it because of the warnings. The problem wasn't with Phoebe. Thanks for your help!
